celery/app/trace.py

```diff
@@ -606,6 +606,8 @@ def _fast_trace_task(task, uuid, request, body, content_type,
     )
     return (1, R, T) if I else (0, Rstr, T)
 
+fast_trace_task = _fast_trace_task  # noqa: E305
```

Comment:
There are only a few references to the underscored name in the unit tests - could we just change the defintion to use the non-underscroed name?